### DISCUSSION TITLE
Make the app responsive

### DISCUSSION THREAD

**Discussion by colebemis** [primer/prism#33] [Ideas]
## Problem

Primer Prism doesn't work on small viewports. Let's make the app usable for small viewports (or at least show a message prompting people to use a larger viewport)

<img width="501" alt="CleanShot 2022-06-14 at 17 06 42@2x" src="https://user-images.githubusercontent.com/4608155/173697380-a4031e51-989f-4fc5-8eb8-eb5dbbcbd685.png">

**Comment by curtisbarnard** [comment]
100% agree. As simple as collapsing that palette menu to the left would make a world of difference.

**Comment by noobyogi0010** [comment]
@curtisbarnard I would like to work on adding responsiveness to the app if no one has already taken it up.
Thanks!
